src/shared/sam/localLambdaRunner.ts

```diff
@@ -66,11 +66,11 @@ function makeResourceName(config: SamLaunchRequestArgs): string {
 
 const SAM_LOCAL_PORT_CHECK_RETRY_INTERVAL_MILLIS: number = 125
 const SAM_LOCAL_PORT_CHECK_RETRY_TIMEOUT_MILLIS_DEFAULT: number = 30000
-const MAX_DEBUGGER_RETRIES_DEFAULT: number = 30
-const ATTACH_DEBUGGER_RETRY_DELAY_MILLIS: number = 200
+const MAX_DEBUGGER_RETRIES_DEFAULT: number = 4
```

Comment:
These retries never happened until it was fixed in https://github.com/aws/aws-toolkit-vscode/pull/1289 . But if it does retry, it is not a great UX (modal dialog). So, don't do 30 of them. Should rarely be needed.